### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sbromberger**
Update `Graphs.jl` examples to `LightGraphs.jl`

### CODE REVIEW

**Comment by sbromberger** [comment]
@simonschoelly  - mind taking a look? I'm not at a system where I can test the LG changes but I think they're correct.

**Comment by sbromberger** [comment]
I didn't do anything that would've affected codecov.

File: README.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ gplot(g)
 
 ## Add node label
 ```julia
-using Graphs
-nodelabel = [1:num_vertices(g)]
+using LightGraphs
+nodelabel = [vertices(g)]
```
Comment (matbesancon) [inline README.md]:
```suggestion
nodelabel = collect(vertices(g))
```

File: README.md [hunk 2]
```diff
@@ -75,13 +75,13 @@ gplot(g, nodelabelsize=nodelabelsize, nodesize=nodesize, nodelabel=nodelabel)
 
 ## Draw edge labels
 ```julia
-edgelabel = [1:Graphs.num_edges(g)]
+edgelabel = [1:ne(g)]
```
Comment (matbesancon) [inline README.md]:
```suggestion
edgelabel = collect(1:ne(g))
```

File: README.md [hunk 3]
```diff
@@ -75,13 +75,13 @@ gplot(g, nodelabelsize=nodelabelsize, nodesize=nodesize, nodelabel=nodelabel)
...
 gplot(g, edgelabel=edgelabel, nodelabel=nodelabel)
 ```
 
 ## Adjust edge labels
 ```julia
-edgelabel = [1:Graphs.num_edges(g)]
+edgelabel = [1:ne(g)]
```
Comment (matbesancon) [inline README.md]:
```suggestion
edgelabel = collect(1:ne(g))
```

File: README.md [hunk 4]
```diff
@@ -121,7 +121,7 @@ gplot(g, layout=spectral_layout)
 ```julia
 nlist = Array(Vector{Int}, 2) # two shells
 nlist[1] = [1:5] # first shell
```
Comment (matbesancon) [inline README.md]:
```suggestion
nlist[1] = collect(1:5) # first shell
```

File: README.md [hunk 5]
```diff
@@ -121,7 +121,7 @@ gplot(g, layout=spectral_layout)
...
-nlist[2] = [6:num_vertices(g)] # second shell
+nlist[2] = [6:nv(g)] # second shell
```
Comment (matbesancon) [inline README.md]:
```suggestion
nlist[2] = collect(6:nv(g)) # second shell
```

File: README.md [hunk 6]
```diff
@@ -121,7 +121,7 @@ gplot(g, layout=spectral_layout)
 ```julia
 nlist = Array(Vector{Int}, 2) # two shells
```
Comment (matbesancon) [inline README.md]:
```suggestion
nlist = Vector{Vector{Int}}(undef, 2) # two shells
```

**Comment by simonschoelly** [comment]
Didn't see this PR, so I made similar changes in #104 . Therefore I am closing this PR now.

**Pull request closed without merging**